llvm/lib/Target/AMDGPU/SIWholeQuadMode.cpp

```diff
@@ -1493,9 +1496,10 @@ void SIWholeQuadMode::lowerCopyInstrs() {
                                 *MRI, MI->getOperand(0)));
     MI->setDesc(TII->get(CopyOp));
   }
+  return !LowerToCopyInstrs.empty();
```

Comment:
Just from visual inspection, should this be `!LowerToMovInstrs.empty() || !LowerToCopyInstrs.empty()`?

Reply:
Thanks, I'll fix this.